identity-android/src/main/java/com/android/identity/SessionEncryption.java

```diff
@@ -184,7 +250,9 @@ void setSendSessionEstablishment(boolean sendSessionEstablishment) {
     }
 
     /**
-     * Encrypts a message to the remote mDL prover.
+     * Encrypts a message to send to a receiver of the opposite type (ie. {@link SessionEncryption}
+     * objects with the ROLE_MDOC role will encrypt messages with the remote mDL prover as the
+     * intended receiver.
      *
```

Comment:
I would move the explanation of `ROLE_MDOC` vs `ROLE_MDOC_READER` up to the constructor (where `role` is passed) and just say _Encrypt a message intended for the remote device_ here.